src/contexts/chain/hooks.tsx

```diff
@@ -216,7 +216,12 @@ export const useMarket = (initalState: ChainState) => {
     }
 
     const [communityPoolCoin] = R.pathOr([], ['communityPool', 0, 'coins'], data).filter((x) => x.denom === chainConfig.primaryTokenUnit);
-    const inflation = R.pathOr(0, ['inflation', 0, 'value'], data);
+    const inflationRates = R.pathOr(0, ['inflation', 0, 'inflation'], data);
+    const ungmInflation = inflationRates.find((item) => {
+      return item.denom === 'ungm';
```

Comment:
can you use the `primaryTokenUnit` inside chainConfig?

Reply:
ok let me check